src/DynamoUtilities/AssemblyConfiguration.cs

```diff
@@ -13,7 +13,7 @@ public sealed class AssemblyConfiguration
 
         public static AssemblyConfiguration Instance
         {
-            get { return instance ?? new AssemblyConfiguration(); }
+            get { return instance ??= new AssemblyConfiguration(); }
```

Comment:
what does this do?

Reply:
??= also assigns right side to the left side.

Reply:
Ugh... Feature 'coalescing assignment' is not available in C# 7.3. Please use language version 8.0 or greater